Bottom: 9dfbd6f5ed32f569db17ac1cb58d3a07f32db0d1
Top:    00ab1f15334092ddeb90e46b0f82ae5698e41f0b
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-01-18 17:58:05 +0800

Refresh of change

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index 2fba3be..e9ea62c 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -163,7 +163,7 @@ struct tun_pcpu_stats {
  * other can only be read while rcu_read_lock or rtnl_lock is held.
  */
 struct tun_file {
-	struct sock sk;
+	  struct sock sk;
 	struct socket socket;
 	struct socket_wq wq;
 	struct tun_struct __rcu *tun;
